src/controllers/simplified-account/settings/webhooks/update/update-webhook.controller.js

```diff
@@ -2,16 +2,72 @@ const { formatSimplifiedAccountPathsFor } = require('@utils/simplified-account/f
 const paths = require('@root/paths')
 const { response } = require('@utils/response')
 const { constants } = require('@govuk-pay/pay-js-commons')
+const webhooksService = require('@services/webhooks.service')
+const { webhookSchema, webhookErrorIdentifiers } = require('@utils/simplified-account/validation/webhook.schema')
+const { validationResult } = require('express-validator')
+const formatValidationErrors = require('@utils/simplified-account/format/format-validation-errors')
+const { responseWithErrors } = require('@controllers/simplified-account/settings/webhooks/create/create.controller')
+const WebhookUpdateRequest = require('@models/webhooks/WebhookUpdateRequest.class')
 
+/**
+ *
+ * @param req {SimplifiedAccountRequest}
+ * @param res
+ * @returns {Promise<void>}
+ */
 async function get (req, res) {
+  const webhook = await webhooksService.getWebhook(req.params.webhookExternalId, req.service.externalId, req.account.id)
+
   response(req, res, 'simplified-account/settings/webhooks/edit', {
+    form: {
+      callbackUrl: webhook.callbackUrl, description: webhook.description, subscriptions: webhook.subscriptions
+    },
     eventTypes: constants.webhooks.humanReadableSubscriptions,
     backLink: formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.webhooks.index, req.service.externalId, req.account.type)
   })
 }
 
-async function post (req, res) {
-  res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.webhooks.index, req.service.externalId, req.account.type))
+/**
+ *
+ * @param req {SimplifiedAccountRequest}
+ * @param res
+ * @param next {function}
+ * @returns {Promise<void>}
+ */
+async function post (req, res, next) {
+  const validations = [
+    webhookSchema.callbackUrl.validate,
+    webhookSchema.description.validate,
+    webhookSchema.subscriptions.validate
```

Comment:
Is there any reason why we want use the callbackUrl, description and subscriptions validations separately as opposed to doing it all in one shot [like this](https://github.com/alphagov/pay-selfservice/blob/e882e378909a4376591e32452503c739b9c74efc/src/controllers/simplified-account/settings/switch-psp/switch-to-worldpay/add-flex-credentials.controller.js#L23)?

Reply:
this is just grouping the validations to be run, they're all run together on line 43